inst/include/Rcpp/sugar/functions/median.h

```diff
@@ -0,0 +1,297 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; tab-width: 8 -*-
+//
+// median.h: Rcpp R/C++ interface class library -- median
+//
+// Copyright (C) 2012 - 2013 Dirk Eddelbuettel and Romain Francois
```

Comment:
2012-2013 makes no sense (and just shows the copy and pasty).  If the file is all yours may as well drop us.  Or keep us in as a courtesy. Your file, your call.